admin_guide/scheduler.adoc

```diff
@@ -215,14 +215,34 @@ Each of the priority function has a weight of '1' applied to it:
 . BalancedResourceAllocation
 . ServiceSpreadingPriority
 
+[[modifying-scheduler-policy]]
+=== Modifying Scheduler Policy
+
+The scheduler policy is defined in a file on the master,
+named *_/etc/origin/master/scheduler.json_* by default,
+unless overridden by the `kubernetesMasterConfig.schedulerConfigFile`
+field in the
+xref:../install_config/master_node_configuration.adoc#master-configuration-files[master configuration file].
+
+To modify the scheduler policy:
+
+. Edit the scheduler configuration file to set predicates and priority functions.
```

Comment:
Should there be an example here? What predicates and priority functions are they setting?